docs/design/api.rst

```diff
@@ -75,3 +75,68 @@ emails, or other method, or ignored.
 * Subscription cancelled.
 
 TODO: figure out how solitude would signal out these events.
+
+Payment method created
+----------------------
+When a new credit card is added event, it’s [immediately visible on the purchase flow](https://cloud.githubusercontent.com/assets/51007/7153010/dcfa48c2-e399-11e4-9607-201c1b630db0.PNG).
+
+There’s no need to send an email unless we’re concerned about unauthorised access. (For example, Facebook would tell you if you’re signed in from an unknown device.)
+
+Payment method updated for subscription
+---------------------------------------
+Gist: each subscription can be paid with different credit cards. When the credit card change, should we send an email that says “This subscription is now paid using B instead of A”?
+
+* If card is updated then immediately used to pay, email receipt will contain [the new card information](https://cloud.githubusercontent.com/assets/51007/7152988/8ffdeb00-e399-11e4-820f-4d6d60c4df01.PNG) in it. No need for separate notification.
+* If card is updated but not used to pay until sometime afterwards, should we let user know? Probably not.
+  * First of all, card is always updated manually, so user always know that it has happened
+  * Secondly, when the email receipt comes a few days/weeks later, it will contain [the new card information](https://cloud.githubusercontent.com/assets/51007/7152988/8ffdeb00-e399-11e4-820f-4d6d60c4df01.PNG) in it
+
+Subscription created
+--------------------
+This notification is already contained within the email receipt. A subscription receipt will have an extra field that says “[Next payment](https://cloud.githubusercontent.com/assets/51007/7153036/4574742c-e39a-11e4-8e90-be13437994c3.PNG)”.
+
+This makes a separate “You’re subscribed to [product]” email unnecessary.
+
+Subscription charge succeeded
+-----------------------------
+The reception of email receipt already indicates that the charge was succeeded. No need to send a separate email.
```

Comment:
Its the same event. When Subscription charge succeeds, we send an email receipt.